ccp/lib/utils/request.py

```diff
@@ -0,0 +1,29 @@
+import requests
```

Comment:
It would be good, if we can add one-line at the start of the file for eg
`"This file handles API requests and returns the response for given API URL along with provided parameters. "`

Reply:
applies to all other files as well.

Reply:
Added